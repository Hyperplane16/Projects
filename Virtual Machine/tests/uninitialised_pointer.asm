FUNC LABEL 0
    MOV REG 0 VAL 50
    MOV STK A REG 0
    MOV PTR B PTR A
    RET
