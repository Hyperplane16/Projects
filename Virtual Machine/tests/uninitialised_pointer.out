Trying to access a pointer that does not exist.
1